civis/resources/_resources.py

```diff
@@ -146,15 +146,20 @@ def deprecated_notice(deprecation_warning):
     return "Deprecation warning!\n------------------\n" + deprecation_warning
 
 
-def doc_from_responses(responses):
+def doc_from_responses(responses, is_iterable):
     """ Return a doc string element from a responses object. The
     doc string describes the returned objects of a function.
     """
     response_code, response_object = next(iter(responses.items()))
     schema = response_object.get('schema', {})
     properties = get_properties(schema)
     if properties:
-        result_doc = "\n".join(docs_from_properties(properties))
+        if is_iterable:
+            resp_type = "civis.response.PaginatedResponse\n"
+        else:
+            resp_type = "civis.response.Response\n"
+        result_doc = resp_type + (
+            "\n".join(docs_from_properties(properties)))
```

Comment:
Now that you're adding the regular/paginated Response class name, I wonder if we need to indent all the docs for the attributes/keys one level "down" (so with four more spaces) to make things look nicer in the compiled docs. What do you think about wrapping them with [textwrap.indent](https://docs.python.org/3/library/textwrap.html#textwrap.indent)?

Reply:
makes sense. actually seems that I can just pass ```level=1``` to ```docs_from_properties```